frontend/dockerfile/dockerfile2llb/convert.go

```diff
@@ -1335,10 +1335,11 @@ func dispatchCopy(d *dispatchState, cfg copyConfig) error {
 	var mode *os.FileMode
 	if cfg.chmod != "" {
 		p, err := strconv.ParseUint(cfg.chmod, 8, 32)
-		if err == nil {
-			perm := os.FileMode(p)
-			mode = &perm
+		if err != nil {
+			return fmt.Errorf("invalid chmod format: '%v'", cfg.chmod)
```

Comment:
Can you use `errors.Errorf` for this one? I think using `pkg/errors` is preferred in this project (as those errors contain additional data that's used for tracing)

Reply:
Thank you.
Based on the above comment, I also tried `errors.Wrap`, but the error message was as below, which didn't seem very useful for me, so I tried using `errors.Errorf`.
```
ERROR: failed to solve: invalid chmod format: strconv.ParseUint: parsing "64a": invalid syntax
```
If the above message is more appropriate, please feel free to let me know.

Reply:
Yeah, I don't think the `ParseUint` error is very useful. Perhaps the error message could mention that it should be a valid octal value, but otherwise it's  probably fine as-is.

Reply:
> the error message could mention that it should be a valid octal 

Oh, certainly!

Furthermore, I have found that if the value is `0o3777777777777`(-1), it is ignored, and otherwise all but the lower 12 bits are meaningless.

I will try to change it again and add more test later.